presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveBasicTableStatistics.java

```diff
@@ -144,6 +171,49 @@ public void testCreatePartitioned()
         }
     }
 
+    @Test
+    public void testCreateExternalPartitioned()
```

Comment:
Since we're doing a lot of back and forth to satisfy every supported metastore version, let's move partitioned tables to a separate PR and focus on unpartitioned for now.